src/steamreport

```diff
@@ -100,26 +132,40 @@ parser.add_argument(
     const=True,
     action="store_const"
 )
+parser.add_argument(
+    "--verbose", "-v",
+    help="Include entire output from data collection.",
+    const=True,
+    action="store_const"
+)
+parser.add_argument(
+    "--force", "-f",
+    help="Skip environment checks and run anyway.",
+    const=True,
+    action="store_const"
+)
 args = parser.parse_args()
 
 # Test environment
-if (not os.environ.get("SNAP")):
-    sys.exit(
-        "Script must be ran from inside a Snap environment.\n" +
-        "Run `snap run --shell steam`, then `steamreport`."
-    )
-
-if (os.system("snapctl is-connected system-observe")):
-    sys.exit(
-        "This script relies on the system-observe Snap plug.\n" +
-        "Outside of the snap, run `snap connect steam:system-observe`."
-    )
-
-if (os.system("snapctl is-connected hardware-observe")):
-    sys.exit(
-        "This script relies on the hardware-observe Snap plug.\n" +
-        "Outside of the snap, run `snap connect steam:hardware-observe`."
-    )
+if (not args.force):
+    if (not os.environ.get("SNAP")):
+        sys.exit(
+            "This script must be ran from inside a Snap environment.\n" +
+            "Run `snap run steam.report`, or\n" +
+            "run `snap run --shell steam`, then `steamreport`."
```

Comment:
I don't think we need to give two ways to run this, just "steam.report" should be enough.